src/libs/TextInputUtils/index.native.js

```diff
@@ -0,0 +1,10 @@
+function getPasswordAutocompleteType() {
+    return 'password';
+}
+
+function setNativePropsWeb() {}
+
+export {
+    getPasswordAutocompleteType,
```

Comment:
Can we just return a string constant instead so we don't have to call a function?

Reply:
Yeah. Sure.